src/features/orbitalfeature.jl

```diff
@@ -34,6 +34,13 @@ function output_shape(ofd::OrbitalFeatureDescriptor, sc::SimpleCodec)
     # return the number
 end
 
+Base.show(io::IO, efd::ElementFeatureDescriptor) = print(io, "OrbitalFeatureDescriptor created.")
+
+function Base.show(io::IO, ::MIME"text/plain", efd::ElementFeatureDescriptor)
+    st = "OrbitalFeatureDescriptor created!"
```

Comment:
I would remove `created!` from this, since the printout is used for inspection after the creation of the object too

Reply:
Done. 👍🏼